ArduPlane/Plane.cpp

```diff
@@ -1013,8 +1013,8 @@ bool Plane::is_taking_off() const
 }
 
 #if HAL_QUADPLANE_ENABLED
-bool Plane::start_takeoff(const float alt) {
-    return plane.quadplane.available() && quadplane.do_user_takeoff(alt);
+bool Plane::start_takeoff(const float alt_m) {
```

Comment:
similar to earlier comments we should consider renaming this to Plane::start_takeoff_m()

Reply:
Ah wait, if we are just renaming the argument to clarify that it is meters and not changing the units then this is fine